xmtp_mls/src/storage/encrypted_store/schema_gen.rs

```diff
@@ -63,6 +63,8 @@ diesel::table! {
         conversation_type -> Integer,
         dm_id -> Nullable<Text>,
         last_message_ns -> Nullable<BigInt>,
+        message_expire_from_ms -> BigInt,
```

Comment:
make sure these are added with the automated script, otherwise we might run into issues with diesel

the command is (from `xmtp_mls` directory)

`cargo run --features="update-schema" update-schema`

Reply:
Yes, I modified them last time manually to check something. After the test generated them via the same command.